specutils/manipulation/replace_segment.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+
+from scipy.interpolate import CubicSpline
+
+from ..spectra import Spectrum1D
+
+
+def spline_replace(spectrum, spline_knots, extrapolation_treatment='data_fill'):
+    """
+    Generates a new spectrum with a region replaced by a smooth spline.
+
+    Parameters
+    ----------
+
+    spectrum : `~specutils.Spectrum1D`
+        The spectrum to be modified.
+
+    spline_knots : array of `~astropy.units.Quantity`
```

Comment:
```suggestion
    spline_knots : 1d `~astropy.units.Quantity`
```
If I'm understanding right?  "array of quantity" sounds like an array of quantity objects rather than a quantity with array-like shape

Reply:
Fixed.